### PR TITLE
[chore] update heroku resource detection info

### PR DESCRIPTION
**Pull request by atoulme**
Move heroku out to its own paragraph and provide a simple example for the heroku detection processor.

### CODE REVIEW

File: processor/resourcedetectionprocessor/README.md [hunk 1]
```diff
@@ -89,17 +89,25 @@ Queries the Docker daemon to retrieve the following resource attributes from the
...
+    override: false
+```
+
 ### Heroku dyno id
 
 In a Heroku application, the [dyno id](https://devcenter.heroku.com/articles/dyno-metadata) is the identifier of the virtualized environment ("dyno") where the application runs.
 
 
-Example:
-
```
Comment (mx-psi) [inline processor/resourcedetectionprocessor/README.md:98]: Looks like this bit is on every other detector, so I think we should add it back?

Comment (atoulme) [reply processor/resourcedetectionprocessor/README.md:98]: the diff is misleading. I am actually adding an example for heroku, see above, the original example for docker is still there.

File: processor/resourcedetectionprocessor/README.md [hunk 2]
```diff
@@ -89,17 +89,25 @@ Queries the Docker daemon to retrieve the following resource attributes from the
...
+
+```yaml
+processors:
+  resourcedetection/docker:
+    detectors: [env, docker]
+    timeout: 2s
+    override: false
+```
+
 ### Heroku dyno id
```
Comment (dmitryax) [inline processor/resourcedetectionprocessor/README.md:102]: ?
```suggestion
### Heroku metadata
```

File: processor/resourcedetectionprocessor/README.md [hunk 3]
```diff
@@ -89,17 +89,25 @@ Queries the Docker daemon to retrieve the following resource attributes from the
...
 
 In a Heroku application, the [dyno id](https://devcenter.heroku.com/articles/dyno-metadata) is the identifier of the virtualized environment ("dyno") where the application runs.
```
Comment (dmitryax) [inline processor/resourcedetectionprocessor/README.md:104]: Can we list all the heroku resource attributes added by the detector?

**Comment by atoulme** [comment]
Closing this PR in favor of #19464, which has the code review changes requested.

**Pull request closed without merging**